server/src/main/kotlin/net/horizonsend/ion/server/features/client/Packets.kt

```diff
@@ -0,0 +1,78 @@
+package net.horizonsend.ion.server.features.client
+
+import io.netty.buffer.Unpooled
+import net.horizonsend.ion.server.miscellaneous.handle
+import net.minecraft.network.FriendlyByteBuf
+import net.minecraft.network.protocol.game.ClientboundCustomPayloadPacket
+import net.minecraft.resources.ResourceLocation
+import net.starlegacy.feature.misc.CustomItems
+import org.bukkit.Bukkit
+import org.bukkit.craftbukkit.v1_19_R2.inventory.CraftItemStack
+import org.bukkit.entity.Player
+import net.horizonsend.ion.server.features.customItems.CustomItems as CustomItems1
+
+enum class Packets(
+	val c2s: (FriendlyByteBuf.(player: Player) -> Unit)? = null,
+	val s2c: (FriendlyByteBuf.() -> Unit)? = null
+) {
+	HANDSHAKE(
+		c2s = {
+			VoidNetwork.modUsers.add(it.uniqueId)
+			PLAYER_ADD.broadcast { writeUUID(it.uniqueId) }
+
+			for (id in VoidNetwork.modUsers - it.uniqueId) {
+				PLAYER_ADD.send(it) { writeUUID(it.uniqueId) }
+			}
+		},
+		s2c = {
+			val items =
+				(
```

Comment:
what unholy mess of formatting happened here

Reply:
idk but it works